packages/chip-input/src/ux-choice-container-attribute.ts

```diff
@@ -0,0 +1,117 @@
+import { UxChoiceAttribute } from './ux-choice-attribute';
+import { customAttribute, bindingMode, bindable, inject, TaskQueue } from 'aurelia-framework';
+
+@customAttribute('ux-choice-container')
+@inject(Element, TaskQueue)
+export class UxChoiceContainerAttribute {
+
+  @bindable({defaultBindingMode: bindingMode.twoWay, primaryProperty: true}) public value: any | any[];
+  @bindable public multiple: boolean | 'auto' = 'auto';
+
+  public isMultiple: boolean = false;
+  private choices: Array<UxChoiceAttribute> = [];
+
+  private isQueued: boolean = false;
+
+  constructor(private element: Element, private taskQueue: TaskQueue) {
+  }
+
+  /* Event passed on the click eventListener */
+  public handleEvent(event: Event) {
+    if (event.target instanceof Element) {
+      const choiceElement: any = event.target.closest('.ux-choice');
+      if (
+        choiceElement !== null &&
+        choiceElement.au !== undefined &&
+        choiceElement.au['ux-choice'] !== undefined &&
+        choiceElement.au['ux-choice'].viewModel instanceof UxChoiceAttribute) {
+        const choice: UxChoiceAttribute = choiceElement.au['ux-choice'].viewModel;
+        this.toggleValue(choice.value);
+      }
+    }
+  }
+
+  /* Callback passed on the TaskQueue when registering child choices */
+  public call() {
+    this.isQueued = false;
+    this.processValue();
+  }
+
+  public bind() {
+    this.multipleChanged();
+    this.valueChanged(this.value);
+  }
+
+  public multipleChanged() {
+    if (this.multiple === 'auto') {
+      this.isMultiple = Array.isArray(this.value) ? true : false;
+    } else if (typeof this.multiple === 'boolean') {
+      this.isMultiple = this.multiple;
+    } else {
+      this.isMultiple = this.multiple === 'multiple' || this.multiple === 'true';
+    }
+    this.element.classList.toggle('ux-choice-container--multiple', this.isMultiple);
+  }
+
+  public attached() {
+    this.element.classList.add('ux-choice-container');
```

Comment:
same with comment in `ux-choice`, if possible, we should add the class as early as possible to avoid unnecessary browser work.